### PR TITLE
Add README.md and /release/ endpoint

### PR DESCRIPTION
**Pull request by Zyko0**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -40,6 +41,48 @@ func main() {
...
+			return
+		}
+
+		reqVersion = sequences[1]
+		if reqArchive == "" || reqVersion == "" {
+			writeShortResponse(res, http.StatusNotFound, "Not found")
+			return
+		}
+
+		archiveUrl := fmt.Sprintf("https://github.com/Scalingo/cli/releases/download/%s/%s", reqVersion, reqArchive)
```
Comment (Soulou) [inline main.go]: Can you write that as a constant?

File: main.go [hunk 2]
```diff
@@ -40,6 +41,48 @@ func main() {
...
+			writeShortResponse(res, http.StatusBadRequest, "Bad request")
+			return
+		}
+		defer githubRes.Body.Close()
+
+		if githubRes.StatusCode != http.StatusOK {
+			writeShortResponse(res, http.StatusBadRequest, "Bad request")
+			return
+		}
+
```
Comment (Soulou) [inline main.go]: It would be appreciated to forward the headers like `Content-Type` and `Content-Length`

Comment (Zyko0) [inline main.go]: How am I supposed to know the length? I'm a bit confused since we are piping the binary

File: main.go (hunk 1 above)
Comment (Zyko0) [inline main.go]: You mean the beginning of the url?
Like `github.com/Scalingo/cli/releases/download`?

Comment (Soulou) [inline main.go]: `https://github.com/Scalingo/cli/releases/download/%s/%s`

File: main.go (hunk 2 above)
Comment (Soulou) [inline main.go]: Github is returning it!

File: main.go (hunk 1 above)
Comment (Soulou) [inline main.go]: And you can call it `githubUrlPattern` for instance

File: main.go (hunk 2 above)
Comment (Soulou) [inline main.go]: We want the headers of githubRes, not req.Header

**Pull request merged**